### PR TITLE
Fixes #16021: Improve ansible technique documentation

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/16021

### CODE REVIEW

File: contrib/rudder-ansible-report/ansible.adoc [hunk 1]
```diff
@@ -1,40 +1,34 @@
-= Ansible report
+= Ansible report plugin
 
 Ansible is an IT automation tool. It can configure systems, deploy software, and orchestrate more advanced IT tasks such as continuous deployments.
+This Rudder Technique aims to integrate the execution and reporting of an Ansible agent in the Rudder one.
 
-== Goal
+Basically, when assigning a Directive declined from this Technique to a node, it will:
```
Comment (amousset) [inline contrib/rudder-ansible-report/ansible.adoc]: no need to capitalize "technique" or "directive"

**Comment by Fdall** [comment]
PR updated with a new commit

File: contrib/rudder-ansible-report/ansible.adoc [hunk 2]
```diff
@@ -1,40 +1,34 @@
...
 
-== Ansible technique Property
+== Usage
 
-=== Pre-requisite
+Make sure that the target node has SSH access to every Ansible targetted hosts.
+The Technique takes two arguments:
 
-This technique will install the necessary Ansible package by generic methods.
+* `PathPlaybookServer` Relative path (relative to */var/rudder/configuration-repository/shared-files/*) to the playbook that will be downloaded on your executioner.
```
Comment (amousset) [inline contrib/rudder-ansible-report/ansible.adoc]: ````for path````

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Pull request merged**